orc8r/cloud/docker/run.py

```diff
@@ -127,6 +137,12 @@ def _parse_args() -> argparse.Namespace:
         help='Stop running containers',
     )
 
+    parser.add_argument(
+        '--clear_db', '-b',
```

Comment:
Nit: convention seems to be `--clear-db`, e.g. from `man grep`. I'm assuming that still translates to `args.clear_db` in python code (?)